charts/gardener-extension-shoot-flux/templates/_helpers.tpl

```diff
@@ -13,3 +13,13 @@ app.kubernetes.io/name
 {{ include "labels.app.key" . }}: {{ include "labels.app.value" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 {{- end -}}
+
+{{- define "image" -}}
+  {{- if eq (printf "%T" .Values.image) "string" }}
```

Comment:
I'd prefer
```suggestion
  {{- if kindIs "string" .Values.image }}
```
https://helm.sh/docs/chart_template_guide/function_list/#kind-functions

Any particular reason you used `printf "%T"`?

Reply:
Nope, I was unaware that helm brings `kindIs`. Neat.